pkg/builder/cproject/builder.go

```diff
@@ -16,37 +16,14 @@ import (
 	"runtime"
 	"strings"
 
+	builder "cbuild/pkg/builder"
 	utils "cbuild/pkg/utils"
 
 	log "github.com/sirupsen/logrus"
 )
 
-type BuilderInterface interface {
-	Build() error
-}
-
-type Builder struct {
-	Runner   utils.RunnerInterface
-	CprjFile string
-	Options  Options
-}
-
-type Options struct {
-	IntDir    string
-	OutDir    string
-	LockFile  string
-	LogFile   string
-	Generator string
-	Target    string
-	Jobs      int
-	Quiet     bool
-	Debug     bool
-	Verbose   bool
-	Clean     bool
-	Schema    bool
-	Packs     bool
-	Rebuild   bool
-	UpdateRte bool
+type CPRJBuilder struct {
```

Comment:
For consistency with devtools [guidelines](https://github.com/Open-CMSIS-Pack/devtools/blob/main/docs/GUIDELINES.md#source-files-cpp-java) should we apply camel case here `CprjBuilder`?

Reply:
Addressed in one of the changes done in [Improvements](https://github.com/Open-CMSIS-Pack/cbuild/pull/29/commits/472704065a8867715b9fefc2c651f85f036e9c02)